settings.mk

```diff
@@ -111,17 +111,21 @@ endif
 # Environment variable OSTYPE is set to cygwin if running under cygwin.
 ifndef CYGWIN
 	OSTYPE?=$(shell echo $$OSTYPE)
-	ifeq ($(OSTYPE),cygwin)
-			CYGWIN:=1
+	ifeq ($(strip $(OSTYPE)), cygwin)
+		CYGWIN:=1
+	else ifeq ($(strip $(OSTYPE)), msys)
+		CYGWIN:=1
 	else
 		CYGWIN:=0
 	endif
 endif
 
+$(info CYGWIN is $(CYGWIN))
 ifeq ($(CYGWIN),1)
 	TEST_ROOT := $(shell cygpath -w $(TEST_ROOT))
 endif
 TEST_ROOT := $(subst \,/,$(TEST_ROOT))
+$(info TEST_ROOT is $(TEST_ROOT))
```

Comment:
I don't think we need to print out TEST_ROOT, it is already printed here: 	https://github.com/adoptium/TKG/blob/15ae4d0746c3fc95564266e156bc57deae8d985a/settings.mk#L276. 
As for `CYGWIN`, if you want need to keep it, please move it to the same place. Using `info` here would cause it to be printed multiple times. Also it will be more meaningful to print something like `$(OSTYPE)` instead of just `CYGWIN is 1`.

Reply:
thanks @renfeiw 
removed both "info" lines